### PR TITLE
Subscribe page layout

### PR DESCRIPTION
**Pull request by paulauus**
Subscription page layout is done, now it just needs to connect to SNS and start communicating with it.
Also some bug fixes and design improvements.

### CODE REVIEW

File: dashboard/pages/Subscribe.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+This page includes the functionality for users to subscribe
+and unsubscribe to email updates about game releases.
+"""
+
+import re
+
+import streamlit as st
+
+# Regex for validating email
+email_pattern = r'^[\w\.-]+@[\w\.-]+\.\w+$'
```
Comment (Leon-JS02) [inline dashboard/pages/Subscribe.py:11]: Neat regex

File: dashboard/pages/Subscribe.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    if st.button("Subscribe"):
+        if re.match(email_pattern, email):
+            st.success(f"You are subscribed with {email}. Bosh!")
+        else:
+            st.warning("Please enter a valid email address.")
+
+st.divider()
+
+cols = st.columns([1, 2, 1])
+with cols[1]:
```
Comment (Leon-JS02) [inline dashboard/pages/Subscribe.py:71]: This block is quite nested, a potential future refactor could change this.

File: dashboard/pages/Subscribe.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    with st.expander("Unsubscribe", expanded=False):
+        unsubscribe_email = st.text_input(
+            "Enter your email:", key="unsubscribe")
+        if st.button("Unsubscribe"):
+            if re.match(email_pattern, unsubscribe_email):
```
Comment (Leon-JS02) [inline dashboard/pages/Subscribe.py:76]: Always good to validate!

**Review by Leon-JS02 — APPROVED** [review]
Nice email validation, good suggestion. Solid dashboarding code.

File: dashboard/pages/Subscribe.py (hunk 2 above)
Comment (paulauus) [reply dashboard/pages/Subscribe.py:71]: Will do with next PR, already fixed.

**Pull request merged**